scripts/segmentation_node

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/home/yemika/Public/anaconda3/envs/tensorflow_env/bin/python
```

Comment:
same problem as in the "cropping" branch, this cannot work

Reply:
you should put the instructions in a txt file not a docx

Reply:
and anyway even if I follow your instructions it will never work, because there is your name in the path.

Reply:
How do you suggest I reform this? Should I include a python environment in the package?

Reply:
I will create a conda virtual env and see how I can modify the shebang

Reply:
Is conda installed on xolotl ? I get "conda: command not found"

Reply:
I guess you need to install it on your xolotl. It is installed on mine. See this link - https://docs.conda.io/projects/conda/en/latest/user-guide/install/linux.html (anaconda installer)

Reply:
You can run `wget https://repo.anaconda.com/archive/Anaconda3-2021.05-Linux-x86_64.sh` in the terminal for downloading the .sh file, then simply `bash Anaconda-latest-Linux-x86_64.sh`.

Reply:
Ok thank you. I know that we already talk about that but your account on Slack has been deactivated so I don't have access to our conversation anymore. Remind me why "#!/usr/bin/python3" didn't work for you ?

Reply:
or **#!/usr/bin/env python3** might work